server/routes/star.js

```diff
@@ -0,0 +1,98 @@
+import { Router } from "express";
+import pool from '../db.js'
+
+const router = Router();
+
+
+router.get("/", async (req, res) => {
+   
+    try {
+        const stars = await pool.query("SELECT * FROM stars");
```

Comment:
This is perfectly fine, however, in large databases we want to be careful with our use of `*` it is always more efficient to explicitly list the columns you wish to return

Reply:
sure.I'm going to implement it.